docs/60-authoring/09-permissions.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Permissions
 ---
-
+// TODO - Well, this is a pretty advance concept in general. I'm not sure how to explain this without kubernetes
```

Comment:
Very advanced. We should scope this down to just Acorn permissions, and then expand in the runtime docs.